lib/builder.js

```diff
@@ -81,8 +83,14 @@ module.exports = class Builder {
   // This method will never throw, and it will never be rejected with anything
   // other than a BuildError.
   build() {
+    if (this._cancelationRequest) {
+      return RSVP.Promise.reject('WTF');
```

Comment:
better error message, likely a `BuildError` with a message similar to`cannot build mid-build`